ansible/roles/fastpath/templates/api-uploader.conf

```diff
@@ -0,0 +1,10 @@
+# OONI API measurement uploader - Python ini format
+# Deployed by ansible, see roles/ooni-backend/templates/api-uploader.conf
+[DEFAULT]
+# arn:aws:iam::676739448697:user/ooni-pipeline, AWS: OONI Open Data
+aws_access_key_id = AKIAJURD7T4DTN5JMJ5Q
+aws_secret_access_key = {{ s3_ooni_open_data_access_key }}
```

Comment:
I think this might have been placed in the vault. I suggest we move it instead into parameter store

Reply:
I can do that, I was using that pattern because it was used in other places for the same key:
https://github.com/ooni/devops/blob/366b6536704b74a68df14b76e4631fb7d769bcca/ansible/host_vars/backend-hel.ooni.org/vars.yml#L1

Should we update those places as well?